tests/issues-sprint-1.coffee

```diff
@@ -20,7 +20,7 @@ describe 'issues-sprint-1', ->
     @timeout(10000)
 
     assert_Weak_Pwd_Fail "", true, ->
-      assert_Weak_Pwd_Fail  "123", false, ->   # this should fail to create an account
+      assert_Weak_Pwd_Fail  "123", true, ->   # this should fail to create an account
```

Comment:
Nice :)

Now you need to move this test to the to the [regression-sprint-1.coffee](https://github.com/TeamMentor/TM_4_0_QA/blob/master/tests/regression-sprint-1.coffee) file